src/main/resources/META-INF/plugin.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <idea-plugin require-restart="true" url="http://tabnine.com">
     <name>Tabnine AI Code Completion- JS Java Python TS Rust Go PHP &amp; More</name>
-    <id>com.tabnine.TabNine</id>
+    <id>com.tabnine.TabNine-Enterprise</id>
```

Comment:
this will always change it, don't we want it only for onprem?

Reply:
@avnerbarr @assapir @amircodota im not sure what to do about this one - I can't change it in runtime... do we want to add a step to the enterprise CI that'll change the id in the plugin.xml?...

Reply:
maybe it is possible to add an if in the XML - https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html

gradle templating and notice that there is a build step `patchPluginXML` 
https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html


https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-patchpluginxml

Reply:
https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-patchpluginxml-pluginxmlfiles